zookeeper-jute/src/main/java/org/apache/jute/BinaryInputArchive.java

```diff
@@ -22,108 +22,124 @@
 import java.io.DataInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
 
 /**
  *
```

Comment:
Yes we have. If we remove this placeholder checkstyle would fail on

>[ERROR] /Users/chenzili/IdeaProjects/zookeeper/zookeeper-jute/src/main/java/org/apache/jute/BinaryInputArchive.java:28: Missing a Javadoc comment. [JavadocType]

However, it just checks whether or not there is a javadoc.

If you want to forbid empty javadoc as well, we can enable 

```
<module name="JavadocStyle">
  <property name="checkEmptyJavadoc" value="true"/>
</module>
```

But I'm afraid it would be not easy to add proper javadoc in this pass. We can file another JIRA fixing all empty javadoc and enable this property. WDYT?

Reply:
Otherwise we can enable it in this pass and set javadoc placeholder. Now I think it would be better. But waiting for your advice.

Reply:
I think we can fix javadocs in a second pass.
This first pass should be as much manual as possible